test/services/helpers/testObjects.js

```diff
@@ -68,19 +68,24 @@ module.exports = (app, opt = {
 		return { team, user };
 	};
 
-	const setupUser = async () => {
-		// create account
-		const user = await users.create();
-		// const account = createTestAccount();
-		// fetch jwt
-		const account = {};
-		const requestParams = {};
-		return { user, account, requestParams };
+	const setupUser = async (userData) => {
+		try {
+			const $user = await users.create(userData);
+			const user = $user.toObject();
+			const requestParams = await login.generateRequestParamsFromUser(user, true);
```

Comment:
Braucht man immer die requestParams? Könnte die Performance verschlechtern

Reply:
An sich nein, aber das als options durchreichen gefällt mir auch nicht. Eventl. sollten wir uns dann noch mal den ganzen workflow zum Nutzer erstellen / jwt erstellen anschauen. Über die neue jwt class könnte man das auch direkt darüber abwickeln. 
Diese müsste dann aber etwas generischer gestaltet werden.
Ein bisschen wurden die testHelpers in diesem PR aufgeräumt, erweitert und das anlegen des Users performanter gemacht. 
Ich finde das reicht erst mal als Verbesserung.